.github/workflows/dapr-base-containers.yaml

```diff
@@ -8,14 +8,18 @@ name: dapr-base-containers
 on:
   repository_dispatch:
     types: [windows-base]
+  pull_request:
+    branches:
+      - master
+      - release-*
 jobs:
   build:
     name: Build base windows images
     runs-on: windows-latest
     env:
       WINDOWS_VERSION: 1809
       TARGET_OS: windows
-      DAPR_REGISTRY: ${{ secrets.DOCKER_REGISTRY }}
+      DAPR_REGISTRY: daprio
```

Comment:
So, why did we change this to be hard coded? This was originally stored in the secrets so that you can build from a fork repo and push to your own registry.

Reply:
I can revert this. The idea was to verify the build in PR.

Reply:
So I added a daprbot target ```/build-windows-base``` to allow iteration during a PR (without adding another 20 minute build job to run on every PR)